css/main.css

```diff
@@ -159,3 +159,45 @@
   max-height: 0;
   overflow: hidden;
   transition: max-height 0.1s ease-out; }
+
+.sponsor-img {
+  width: 100px;
+  height: 100px;
+  position: relative;
```

Comment:
i couldn't achieve a smooth transition from right edge of the screen to the left without making this absolute, but then images overlap obv. thoughts?

Reply:
ahh wait yeah this is hard LOL ill take a look into it today and see if i can figure it out